pilot/util/harvester.py

```diff
@@ -282,10 +310,17 @@ def publish_job_report(job, args, job_report_file="jobReport.json"):
                 if 'logfileReport' in executor:
                     executor['logfileReport'] = {}
 
-        write_json(dst_file, job_report)
+        if write_json(dst_file, job_report):
+            return True
+        else:
+            return False
 
     except IOError:
         logger.error("job report copy failed")
+        return False
+    #except:
```

Comment:
Remove these lines. Flake8 has a bug that complains also about comments..

Reply:
done